ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision
+    post:
+    - chain: cucushift-installer-rehearse-azure-ipi-deprovision
```

Comment:
We need to deprovision the created minimal permission sp, right?

Reply:
good catch, fixed.